switch_model/transmission/transport/build.py

```diff
@@ -193,18 +193,25 @@ def define_components(mod):
         'trans_length_km', 'trans_efficiency', 'existing_trans_cap')
     mod.trans_new_build_allowed = Param(
         mod.TRANSMISSION_LINES, within=Boolean, default=True)
+    mod.trans_capital_cost_per_mw_km = Param(
+        within=NonNegativeReals,
+        default=1000)
     mod.TRANS_BLD_YRS = Set(
         dimen=2,
         initialize=mod.TRANSMISSION_LINES * mod.PERIODS,
-        filter=lambda m, tx, p: m.trans_new_build_allowed[tx])
+        filter=lambda m, tx, p: m.trans_new_build_allowed[tx] and m.trans_capital_cost_per_mw_km != float("inf"))
     mod.BuildTx = Var(mod.TRANS_BLD_YRS, within=NonNegativeReals)
-    mod.TxCapacityNameplate = Expression(
+    mod.NewTxCapacity = Expression(
```

Comment:
Is there a reason why you removed Nameplate? I think we can still add it for more readability!

Reply:
I didn't remove Nameplate, it's still there just further down in the code. Git diff is just being a bit weird.

Reply:
Ok. So that means that the name of this expression is NewTxCapacityNameplate? or did you just moved that to another expression?

Reply:
Not sure what you mean. There use to be and still is a `TxCapacityNameplate`. What I've done is added a `NewTxCapacity` expression.